Tests/NetworkRequestTests.swift

```diff
@@ -104,4 +104,61 @@ class NetworkRequestTests: XCTestCase {
         XCTAssertEqual(urlRequest.cachePolicy, cachePolicy, file: file, line: line)
         XCTAssertEqual(urlRequest.timeoutInterval, timeout, file: file, line: line)
     }
+    
+    func test_NetworkRequest_InitEmptyRequest() {
```

Comment:
Small typo on the method name I think:
```
func test_NetworkRequest_RemptyResponseInit() {
```

Reply:
I am confused by this one. What "Rempty"?

Reply:
I think you meant to put 
```
func test_NetworkRequest_EmptyResponseInit()
```